### PR TITLE
Fix wrong commands path on Makefile

### PR DESCRIPTION
**Pull request by ajnavarro**
Signed-off-by: Antonio Jesus Navarro Perez <antnavper@gmail.com>

### CODE REVIEW

**Comment by bzz** [comment]
Tried locally, `make packages` produces the executable binaries now 🎉 

Seems a bit confusing due to https://github.com/src-d/guide/blob/master/engineering/conventions-go.md#executables

**Comment by erizocosmico** [comment]
Shouldn't we instead change the folder name to `cmd` as the engineering guide says?

File: cmd/gitbase/main.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"os"
 
-	"github.com/src-d/gitbase/cli/gitbase/cmd"
+	"github.com/src-d/gitbase/cmd/gitbase/cmd"
```
Comment (erizocosmico) [inline cmd/gitbase/main.go]: maybe we should move the contents of the cmd folder into the gitbase folder. It does not make much sense to have them in a folder and the path looks weird

**Comment by ajnavarro** [comment]
@erizocosmico done.

File: cmd/gitbase/server.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package cmd
+package main
```
Comment (smola) [inline cmd/gitbase/server.go]: Why? Subcommnads are supposed to be implemented in a subpackage.

Comment (erizocosmico) [reply cmd/gitbase/server.go]: Why do they need to be in a subpackage?

Comment (mcuadros) [reply cmd/gitbase/server.go]: Testing.

**Comment by mcuadros** [comment]
Who this PR should the problem of the build?

**Comment by ajnavarro** [comment]
@mcuadros Packfile had a wrong path pointing to the commands folder.

**Comment by mcuadros** [comment]
But this PR is not changing the Makefile

**Comment by ajnavarro** [comment]
That was the first approach, but we decide to do it correctly and change instead the folder to the correct one due to the company guidelines: `cli` to `cmd`.

**Comment by mcuadros** [comment]
But this PR is removing the package for the commands.

**Comment by ajnavarro** [comment]
Moved all commands to `cmd/gitbase/command` package after an offline talk with @mcuadros.

**Comment by ajnavarro** [comment]
Merging to fix the build. If we want to add more changes to the path, we can do it into another PR.

**Pull request merged**